lib/couchbase-orm/railtie.rb

```diff
@@ -46,11 +46,10 @@ def self.rescue_responses
             # Initialize Couchbase Mode. This will look for a couchbase.yml in the
             # config directory and configure Couchbase connection appropriately.
             initializer 'couchbase.setup_connection' do
-                config_file = Rails.root.join('config', 'couchbase.yml')
-                if config_file.file? &&
-                    config = YAML.load(ERB.new(File.read(config_file)).result)[Rails.env]
-                    ::CouchbaseOrm::Connection.options = config.deep_symbolize_keys
-                end
+                ENV['COUCHBASE_BUCKET'] ||= Rails.application.config_for(:couchbase).bucket
+                ENV['COUCHBASE_CONNECTION_STRING'] ||= Rails.application.config_for(:couchbase).connection_string
+                ENV['COUCHBASE_USER'] ||= Rails.application.config_for(:couchbase).username
+                ENV['COUCHBASE_PASSWORD'] ||= Rails.application.config_for(:couchbase).password
```

Comment:
Is it more in ruby best practice?

Reply:
I'm not really sure of this.
It could be great to have the review of old rubyists

Reply:
I don't think code should change env variables

Reply:
ok let's me try a better implem.